framework/PostProcessors/LimitSurfaceIntegral.py

```diff
@@ -268,39 +275,35 @@ def collectOutput(self, finishedJob, output):
     evaluation = finishedJob.getEvaluation()
     if isinstance(evaluation, Runners.Error):
       self.raiseAnError(RuntimeError, "No available output to collect (run possibly not finished yet)")
+    pb = evaluation[1]
+    lms = evaluation[0][0]
+    if output.type == 'PointSet':
+      # we store back the limitsurface
+      dataSet = lms.asDataset()
+      loadDict = {key: dataSet[key].values for key in lms.getVars()}
+      loadDict[self.computationPrefix] = np.full(len(lms), pb)
+      output.load(loadDict,'dict')
+    # NB I keep this commented part in case we want to keep the possibility to have outputfiles for PP
```

Comment:
remove commented lines

Reply:
I would like to keep it because (as I wrote)
```NB I keep this commented part in case we want to keep the possibility to have outputfiles for PP```